### PR TITLE
Feature/1559 corp profile setup

### PR DESCRIPTION
**Pull request by eduardosmoura**
@designhorf I need your help once more buddy fixing the member form, please. I will take of all handlers and actions.

### CODE REVIEW

**Comment by designhorf** [comment]
@eduardosmoura polished the form and created custom radio buttons. Please check it out

**Comment by eduardosmoura** [comment]
Awesome @designhorf! You are the man!

**Comment by andregoncalves** [comment]
@eduardosmoura can you please add the UI screen to storybook.

**Comment by eduardosmoura** [comment]
Will do @andregoncalves.

**Comment by eduardosmoura** [comment]
@andregoncalves UI added to the storybook.

**Comment by andregoncalves** [comment]
Is it complete? Can i merge?

**Comment by eduardosmoura** [comment]
The UI is complete @andregoncalves

File: .gitignore [hunk 1]
```diff
@@ -25,6 +25,7 @@ dist/
 src/renderer/angular/app.templates.js
 static/stylesheets/css/main.css
 static/stylesheets/css/*
+storybook-static/
```
Comment (andregoncalves) [inline .gitignore]: Is this windows specific?

File: src/main/main-window.js [hunk 2]
```diff
@@ -51,7 +51,7 @@ export const createMainWindow = () => {
 
 	mainWindow.loadURL(webAppPath);
 
-	mainWindow.setMenu(null); // in order to don't show electron default menu bar
+	// mainWindow.setMenu(null); // in order to don't show electron default menu bar
```
Comment (andregoncalves) [inline src/main/main-window.js]: Why is this?

File: src/renderer/wallet/main/index.jsx [hunk 3]
```diff
@@ -225,6 +226,10 @@ class Main extends Component {
 						path={`${match.path}/create-corporate-profile`}
 						component={CorporateWizardContainer}
 					/>
+					<Route
```
Comment (andregoncalves) [inline src/renderer/wallet/main/index.jsx]: This is will probably needs refactoring later, to move everything corporate related to the /corporate path

File: .gitignore (hunk 1 above)
Comment (eduardosmoura) [reply .gitignore]: That folder appeared when I ran the build storybook command and I didn't want to add generated files to the repository.

File: src/main/main-window.js (hunk 2 above)
Comment (eduardosmoura) [reply src/main/main-window.js]: Ops, my bad, reverting that change.

File: src/renderer/wallet/main/index.jsx (hunk 3 above)
Comment (eduardosmoura) [reply src/renderer/wallet/main/index.jsx]: Sure thing.

**Comment by eduardosmoura** [comment]
@andregoncalves changes resolved.

File: .gitignore (hunk 1 above)
Comment (andregoncalves) [reply .gitignore]: Ok. must be a windows specific thing for storybook.

**Pull request merged**